core-plugins/docs/FileMove-action.md

```diff
@@ -0,0 +1,52 @@
+# File Move Action
+
+
+Description
+-----------
+Moves a file or files.
+
+
+Use Case
+--------
+This action can be used when a file or files need to be moved to a new location in an HDFS cluster, 
```

Comment:
should we remove `hdfs` ?

Reply:
might be good to say the source and destination path should be in the same filesystem.

Reply:
ah yes, missed this, will fix.